Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN        bash /tmp/env-config.sh
 # to be limited only by the Calypso build speed.
 COPY       ./package.json ./npm-shrinkwrap.json /calypso/
 RUN        true \
+           && npm install -g npm@6.1.0 \
```

Comment:
Ideally we'd run this independently and earlier than the full deps install. This particular step should be long-lived and reused across many more builds than the rest of the npm dependency install.

Optionally, you might merge #25372 here.

Reply:
merged #25372! much better solution.